core/tests/CMakeLists.txt

```diff
@@ -34,5 +35,6 @@ endif()
 
 if(${run_e2e_tests})
     add_subdirectory(gateway_e2e)
+    add_subdirectory(performance_e2e)
```

Comment:
>add_subdirectory(performance_e2e) [](start = 4, length = 33)

I think we should separate the performance_e2e from the e2e test, don't you think? have it's own flag run_performance_tests